runtime/src/main/scala/akka/grpc/scaladsl/GrpcExceptionHandler.scala

```diff
@@ -21,6 +21,11 @@ object GrpcExceptionHandler {
   private val INVALID_ARGUMENT = Trailers(Status.INVALID_ARGUMENT)
   private val UNIMPLEMENTED = Trailers(Status.UNIMPLEMENTED)
 
+  /**
+   * Helper method to access an Akka internal method (Until Akka 2.6.5).
+   */
+  def toClassic(system: ClassicActorSystemProvider): ActorSystem = system.classicSystem
```

Comment:
Should we mark this helper 'internal'? I'd say when 2.6.5 has been out for a while we can eventually drop it, and specify when on 2.6 you need at least 2.6.5.

Reply:
Added a note in the comment.